diff -Nrup a/src/idomediaplayermenuitem.c b/src/idomediaplayermenuitem.c
--- a/src/idomediaplayermenuitem.c	2019-12-04 21:44:50.000000000 +0100
+++ b/src/idomediaplayermenuitem.c	2020-06-10 13:51:06.456373431 +0200
@@ -81,7 +81,7 @@ ido_media_player_menu_item_draw (GtkWidg
       int y;
 
       gtk_style_context_get_color (gtk_widget_get_style_context (widget),
-                                   gtk_widget_get_state (widget),
+                                   gtk_widget_get_state_flags (widget),
                                    &color);
 
       gtk_widget_get_allocation (widget, &allocation);
